### DISCUSSION TITLE
Cookies and file protocol with local HTML file

### DISCUSSION THREAD

**Discussion by mirsylvain** [react-native-webview/react-native-webview#2590] [Q&A]
HI everyone,

I used the ways decribed in documentation to open a local HTML file in Android (file:///android_asset) with my bundled app in js.

It seems that cookies are not working at all doing this.
Is there any way to use cookies even if I open a local HTML file this way in a Webview component ?

**Comment by yusera-ansari** [comment]
hey! were you able to solve this?
